src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/coreperson/CorePersonMigrationService.kt

```diff
@@ -121,19 +114,19 @@ class CorePersonMigrationService(
   }
 }
 
```

Comment:
Ive just commented out stuff that no longer compiles for now rather than completely delete - something similar may be needed if we go ahead with a migration